### ISSUE TITLE
error when using in tests with jest

### ISSUE THREAD

**Issue by szamanr** [minwork/react#8]
hi! after upgrading from v2.0.3 to v3.1.0, i get this error in one of my test suites:
```
PointerEvent is not defined
ReferenceError: PointerEvent is not defined
    at PointerEvent (/home/r/workspace/perdoo/frontend/node_modules/use-long-press/index.js:1:534)
```

i'm using the library only in one place, like this:
```
  const toggleDevModeWhenLongPressedThreeTimes = useLongPress(toggleDevMode, {
    onCancel: switchBetweenDevModes,
  });
```

i'm using jest with testing-library in a react + next.js project. here's my relevant dependencies:
```
"@testing-library/dom": "8.19.0",
"@testing-library/jest-dom": "5.16.5",
"@testing-library/react": "13.4.0",
"@testing-library/user-event": "13.5.0",

"jest": "29.4.3",
"next": "13.2.4",
"react": "18.2.0",
"use-long-press": "3.1.0",
```

multiple tests fail in one spec file. nothing special, just clicks a button, not even using long press here so not sure what's happening. here's one of the tests:

```
it("can export and print", async () => {
  await renderGroupProfile();
  const exportBtn = screen.getByText(/file_download/i);
  userEvent.click(exportBtn);
  expect(screen.getByText(/export as pdf/i)).toBeInTheDocument();
  expect(screen.getByText(/print full page/i)).toBeInTheDocument();
});
```

**Comment by minwork** [comment]
Hey, yes, that's because jsdom do not support PointerEvents yet (see: jsdom/jsdom#2666). Internally I am using polyfill for tests (you can find it [here](https://github.com/minwork/react/blob/main/packages/use-long-press/src/lib/tests/pointer-event.polyfill.ts)) which needs to be imported before all tests.

I will check if I can refactor hook in a way that won't trigger this error in tests while using it as external library.

**Comment by minwork** [comment]
@szamanr could you try `use-long-press@3.1.1-alpha.1` and see if it solves that problem?

**Comment by szamanr** [comment]
@minwork it did, thank you!

**Comment by minwork** [comment]
Published fix in `v3.1.1`
